cmd/operator/main.go

```diff
@@ -202,6 +209,28 @@ func run(fs *flag.FlagSet) int {
 		return 1
 	}
 
+	if ok := operator.HasFeatureGate("auto-gomemlimit"); ok {
```

Comment:
```suggestion
	if featureGates.Map["auto-gomemlimit"] {
```

Reply:
Thank for your review.  Change to `(*featureGates.Map)["auto-gomemlimit"]`